Adds link to the temporary alpha docs

peteratticusberg opened the pull request:
Adds a link to the alpha docs in the alpha read me, to my knowledge it's not publicly listed anywhere ¯\_(ツ)_/¯

oliviertassinari commented:
> to my knowledge it's not publicly listed anywhere 

Yes, it's: https://github.com/callemall/material-ui/blob/master/ROADMAP.md#next. I think that it's a good idea to add it in the homepage. Still, we could give a bit of context.

oliviertassinari commented:
Ohhh, I think that PR was targeting the master branch

peteratticusberg commented:
I didn't replace the existing links because I figured that by the time we launched the alpha material-ui.com would be the host (rather than the temporary firebase url). 

Also, there are some pages that exist on the master documentation website that don't exist on alpha documentation site (e.g. Require Knowledge). 

That being said I did just go ahead and replace the very first link (to the "documentation site") and removed the link I added, I think that makes slightly more sense. We'll have to remember to change it back to material-ui.com though once we switch that dns for that over.

oliviertassinari commented:
> We'll have to remember to change it back to material-ui.com though once we switch that dns for that over.

I don't think that it's an issue as we already have to do it for different other locations.

peteratticusberg commented:
👍
